public class org.bouncycastle.util.test.TestFailedException extends java.lang.RuntimeException {
  private org.bouncycastle.util.test.TestResult _result;

  public org.bouncycastle.util.test.TestFailedException(org.bouncycastle.util.test.TestResult);
    Code:
       0: aload_0
       1: invokespecial #15                 // Method java/lang/RuntimeException."<init>":()V
       4: aload_0
       5: aload_1
       6: putfield      #14                 // Field _result:Lorg/bouncycastle/util/test/TestResult;
       9: return

  public org.bouncycastle.util.test.TestResult getResult();
    Code:
       0: aload_0
       1: getfield      #14                 // Field _result:Lorg/bouncycastle/util/test/TestResult;
       4: areturn
}
